tests/phpunit/tests/rest-api/rest-posts-controller.php

```diff
@@ -1527,6 +1528,32 @@ public function test_get_items_not_sticky_with_exclude_no_sticky_posts() {
 		$this->assertPostsWhere( " AND {posts}.ID NOT IN ($id3) AND {posts}.post_type = 'post' AND (({posts}.post_status = 'publish'))" );
 	}
 
+	public function test_get_items_with_custom_search_columns() {
```

Comment:
Can we have a docblock with a description, `@ticket` annotation, and if suitable, a `@covers WP_REST_Posts_Controller::get_items` annotation here? 🙂

Reply:
Added.